### DISCUSSION TITLE
vite watch is not working in node_modules directory.

### DISCUSSION THREAD

**Discussion by keymasroy** [nuxt/nuxt#16131] [Questions]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.15.1`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `ssr`, `modules`, `css`, `experimental`, `vite`, `hooks`, `ustra`
- Runtime Modules:  `@ustra/nuxt@3.0.0-pr.21`, `@hcorestore/common@1.0.0`
- Build Modules:    `-`
------------------------------

### Reproduction

later...

### Describe the bug

I want to watch a directory in node_modules and set nuxt configuration as below.

```typescript
export default defineNuxtConfig({
  ssr: false,
  modules: ['@ustra/nuxt', '@hcorestore/common'],
  experimental: {
    payloadExtraction: false,
  },
  vite: {
    server: {
      watch: {
        ignored: ['!**/node_modules/@ustra/**'],
      },
    },
    optimizeDeps: {
      exclude: ['@ustra/nuxt', '@ustra/core'],
    },
  },
})
```

but is not working...
Anyone has solution ????

**Comment by harlan-zw** [comment]
Maybe just symlink it?
